analyzer/tests/functional/fixit/test_fixit.py

```diff
@@ -211,7 +212,9 @@ def test_fixit_file_modification(self):
             'Skipped files due to modification since last analysis',
             err)
 
-        pathlib.Path(source_file_cpp).touch()
+        date = datetime.datetime.now() + datetime.timedelta(hours=1)
```

Comment:
Could you please add a comment above this line why do we need to use this instead of touch function. This way we will not forget this.